src/content/docs/journey-migration/migrating-apm.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Install APM agents"
```

Comment:
Same feedback as infra; this title is ambiguous/duplicative-looking in google or elsewhere.

Reply:
Fixed!